### DISCUSSION TITLE
Shop Icon

### DISCUSSION THREAD

**Discussion by jcbjoe** [twilio-labs/paste#2962] [Request: Icon] [answered]
Hi,

I would like to request the Shop icon to be added to Paste. We have a marketplace inside of Twilio flex and this icon is the most relevant for this.

Streamline Link: https://www.streamlinehq.com/icons/streamline-light

Thanks,
Joe

**Comment by loreina** [comment]
Hi @jcbjoe, could you share a screenshot of where the icon would be used? And do you mind sharing a direct link to the icon you're looking for? The link directed to the entire Streamline Light library. Thanks!

**Comment by jcbjoe** [comment]
Hi @loreina!

Thanks for getting back to me! Apologies for the invalid link

Here is where the icon will be used:
![image](https://user-images.githubusercontent.com/3605644/213661427-824d4349-c677-44cd-acbd-81829f40102c.png)
In place of the book icon at the bottom of the sidebar

Here is a picture
![image](https://user-images.githubusercontent.com/3605644/213656017-4e9a1fbf-9683-41c3-b1c7-ed884198e0f0.png)

The icon is called "shop" in the "streamline-light" pack. https://www.streamlinehq.com/icons/streamline-light?search=shop

It wont let me copy the exact logo so I will send a screenshot too
![image](https://user-images.githubusercontent.com/3605644/213656398-3b567cf8-3023-49c8-9fec-2185c33f02ca.png)

Thanks,
Joe

**Comment by loreina** [comment]
Hi Joe,

Thanks for the added context and links. I brought this back to the team and we think this would be a great icon addition. We'll follow up in this discussion once the icon's been added to Paste.

**Reply by jcbjoe** [reply]
Amazing! Thank you. Excited to see the shop icon in paste!

**Comment by nkrantz** [comment, marked as answer]
Hey @jcbjoe, feel free to follow along with this PR (https://github.com/twilio-labs/paste/pull/3095). When it merges, all of the new icons will be available for use!
